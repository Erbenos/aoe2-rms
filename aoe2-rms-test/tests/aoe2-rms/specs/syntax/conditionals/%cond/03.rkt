#lang aoe2-rms
; Set up a NOT conditional - place a cow when "infinite resources" is not true.

<OBJECTS-GENERATION>
(%cond ['INFINITE_RESOURCES]
       [#f (create-object 'DLC_COW
                          (set-place-for-every-player)
                          (find-closest))])
