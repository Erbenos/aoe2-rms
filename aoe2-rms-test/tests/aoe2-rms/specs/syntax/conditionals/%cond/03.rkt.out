<OBJECTS_GENERATION>
if INFINITE_RESOURCES
else
  create_object DLC_COW {
    set_place_for_every_player
    find_closest
  }
endif
